### DISCUSSION TITLE
Error: failed to initialize repository resources

### DISCUSSION THREAD

**Discussion by DhamodharReddy** [argoproj/argo-cd#17917] [Q&A]
I wanted to bring up an issue I've encountered with the Argocd application I've been working on. Everything was working perfectly on Argocd for a while, but lately getting the error. This error is only for few applications.
Whenever I try to sync the application, getting below error.

**_" Failed to load target state: failed to generate manifest for source 1 of 1: rpc error: code = Unknown desc = failed to initialize repository resources: rpc error: code = Internal desc = Failed to fetch default: git fetch origin --tags --force --prune failed timeout after 1m30s."_**

I haven't found a solution yet. Any insights or suggestions on how to resolve this?

**Comment by vildand** [comment]
see [#21054](https://github.com/argoproj/argo-cd/discussions/21054)
